README.md

```diff
@@ -1,6 +1,16 @@
 # dp-legacy-cache-api
 
-REST API for managing cache control information for pages within the legacy CMS
+REST API for managing cache control information for pages within the legacy CMS. dp-legacy-cache-api is called by:
+- dp-legacy-cache-proxy to read the release time for a particular item of content
+- Zebedee to set the right cache time for content based on publish notifications
+
+| Database Fields              | 
```

Comment:
It's not clear at this point that "the api talks to documentdb/mongodb(?), in the _<name_of_collect>_ collection" so might be worth mentioning that.

- these fields are in documentdb or mongodb collections (which collection?), running in the env.?
- what does `ID` in the next line refer to - ID of _what_, exactly (page? release?)?
- `path` refers to the path (URI) of the published page?